capabilities.json

```diff
@@ -313,6 +313,13 @@
                         "bool": true
                     }
                 },
+                "displayAll": {
+                    "displayName": "Display all",
+                    "displayNameKey": "Visual_DisplayAll",
+                    "type": {
+                        "bool": true
```

Comment:
Please do not use "true" value by default for your setting cause this can affect to already created users reports